src/boot/efi/stub.c

```diff
@@ -110,6 +110,10 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
                 efivar_set(L"LoaderImageIdentifier", loaded_image_path, FALSE);
         }
 
+        /* same as above with LoaderInfo */
+        if (efivar_get_raw(&global_guid, L"LoaderInfo", &b, &size) != EFI_SUCCESS)
+                efivar_set(L"LoaderInfo", L"systemd-stub " PACKAGE_VERSION, FALSE);
```

Comment:
hmm, should we really do that? I mean, the stub is the stub and the loader is something before the stub. If sd-boot is used we after all will go through both the stub and the loader, and should probably report both, instead of just one...

Maybe add a new variable for that instead, and then pick up both in bootctl?